met-cron/sample.env

```diff
@@ -1,20 +1,96 @@
+# GDX MET CRON Configuration
+# For more information on these values, please see the documentation
+# or met-cron/config.py
 
-DATABASE_USERNAME=analytics
-DATABASE_PASSWORD=analytics
-DATABASE_NAME=met
-DATABASE_HOST=localhost
-DATABASE_PORT=5432
+# Changes Flask's run mode and the set of env vars are used to configure the app. You should not need to change this here.
+FLASK_ENV=development
 
-MET_DATABASE_USERNAME=met
-MET_DATABASE_PASSWORD=met
-MET_DB_NAME=met
-MET_DATABASE_HOST=localhost
-MET_DATABASE_PORT=5432
+USE_DEBUG=True # Enable a dev-friendly debug mode
+TESTING= # Handle errors normally (False) or raise exceptions (True)
 
+# Miscellaneous Settings
+SECRET_KEY="" # For Flask sessions. If unset, this value is randomized
+# disables certain checks for user permissions and tenant access. Buggy.
```

Comment:
Does this comment mean that when you disable certain checks for user permissions and tenant access that unexpected behaviour may occur? Why would a developer want to disable those checks in the first place?

Reply:
I removed this comment as it's not applicable to our scenario. The checks in this case are associated with the tenant. In a multi-tenant environment like ours, we verify if this variable is set to false. In such cases, we interpret the application as multi-tenant, and consequently, we assess user access to a specific tenant to restrict engagement access within that tenant. Conversely, if this variable is set to true, we consider it a single-tenant environment, eliminating the need to check user tenant access.